### PR TITLE
SPCTR-3406 [Container] Padding should inherit from desktop

### PR DESCRIPTION
**Pull request by adi3890**
### Description
- Removed defaults from responsive devices.
- inherits values from desktop if empty.

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

**Review by rahulvarma722 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Improvement: Container - Updated padding settings so responsive devices takes default value from desktop css.
```
Comment (bsf-zanev) [inline README.md:219]: Let's have something like this?

```suggestion
* Improvement: Container - Updated responsive padding settings to inherit from bigger devices by default.
```

File: readme.txt [hunk 2]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 == Changelog ==
 
 = x.x.x =
+* Improvement: Container - Updated padding settings so responsive devices takes default value from desktop css.
```
Comment (bsf-zanev) [inline readme.txt:219]:
```suggestion
* Improvement: Container - Updated responsive padding settings to inherit from bigger devices by default.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update required.

File: README.md (hunk 1 above)
Comment (adi3890) [reply README.md]: updated

File: readme.txt (hunk 2 above)
Comment (adi3890) [reply readme.txt]: updated

**Review by bsf-zanev — APPROVED** [review]

**Pull request merged**